<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-image: url(C:/Projects/Python/Course/Course/assets/militarybcnd.png)

}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;; color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Таблица миссий</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>456</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:50%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>550</y>
      <width>691</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="bot">
     <item>
      <widget class="QPushButton" name="back">
       <property name="minimumSize">
        <size>
         <width>341</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>341</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run_mission">
       <property name="minimumSize">
        <size>
         <width>341</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>341</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Выполнить миссию</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>653</width>
      <height>223</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="top">
     <item>
      <widget class="QTableView" name="missions">
       <property name="minimumSize">
        <size>
         <width>651</width>
         <height>221</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777213</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="placeholder.qrc"/>
 </resources>
 <connections/>
</ui>
